src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/FormPass.php

```diff
@@ -53,11 +54,15 @@ public function process(ContainerBuilder $container)
         $typeExtensions = array();
 
         foreach ($container->findTaggedServiceIds('form.type_extension') as $serviceId => $tag) {
-            $alias = isset($tag[0]['alias'])
-                ? $tag[0]['alias']
-                : $serviceId;
+            $extendedType = $serviceId;
+            if (isset($tag[0]['extended_type'])) {
```

Comment:
I'm not a fan of this renaming:

- it does not really bring anything new
- the new name will not look good in XML configuration files, as underscores in attribute names is not common at all. The current config looks better.